cmd/oras/internal/option/target.go

```diff
@@ -154,7 +154,7 @@ func (opts *Target) NewReadonlyTarget(ctx context.Context, common Common, logger
 		}
 		info, err := os.Stat(opts.Path)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("invalid path in %q: %w", opts.RawReference, err)
```

Comment:
`os.Stat` failure does not imply `invalid path`. Can you try [`fs.ErrNotExist`](https://pkg.go.dev/io/fs#pkg-variables) for a better error message handling?